dll/win32/newdev/newdev.c

```diff
@@ -24,6 +24,7 @@
 
 #include <stdio.h>
 #include <winnls.h>
+#include <assert.h>
```

Comment:
```suggestion
#include <assert.h>
#include <stdio.h>
#include <winnls.h>
```

Reply:
- Comments 2 - 3 - 4 : Ok, done + include<stdlib> for definition of _countof
- Comment1 : I don't understand. I'm agree that the test is useless, it is why I removed it and moved earlier. I have replaced it by an assert because now, it should never happens.
- Comment 5 : What is the difference ?

Reply:
> What does Windows do?

I feel like windows XP SP3 doesn't do any recursive searching at all. And it just tests for the presence of an inf file without even opening it.